run_tests.sh

```diff
@@ -125,12 +113,10 @@ function run_tests()
   # instead of tearing down the container environment after each integration test,
   # we will tear it down only after all tests have ran. This optimizes significant
   # amount of time for the integration tests.
-  if [[ "${integration_tests_setup}" == 1 ]]; then
-    printf '\n' # add new line after the last "OK"
-    say 'Tearing down containers used in integration tests...'
-    teardown_containers
-    printf '\n'
-  fi
+  printf '\n' # add new line after the last "OK"
+  say 'Tearing down containers used in integration tests...'
+  teardown_containers
+  printf '\n'
```

Comment:
Can we try moving the first `printf` into the `say` function? Also, consider moving the last `printf` to the `teardown_containders`  function.